### PR TITLE
PMM-6073 qan usage 2

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Recovered. qan-intro is now a one-page resource. Other pages are redirected to this one.

### CODE REVIEW

**Comment by PaulJacobs-percona** [comment]
See https://new.percona.com/doc/percona-monitoring-and-management/2.x/qan-intro.html

File: source/qan-intro.rst [hunk 1]
```diff
@@ -1,33 +1,233 @@
...
+
+The *Details* tab contains a *Query time distribution* bar (only for MySQL databases)
+and a set of *Metrics* in collapsable subpanels.
+
+.. image:: /_images/PMM_Query_Analytics_details.jpg
+
+- The *Query time distribution* bar shows a query's total time made up of colored segments, each segment representing the proportion of time spent on one of the follow named activities:
+
+  - ``query_time``: Elapsed time between receiving a request and the returned result.
+  - ``lock_time``
```
Comment (PaulJacobs-percona) [inline source/qan-intro.rst:163]: @lunaticusgreen Where can I find descriptions for these?

**Comment by rnovikovP** [comment]
@PaulJacobs-percona  can we deploy this on the internal website for a review?

**Comment by PaulJacobs-percona** [comment]
> @PaulJacobs-percona can we deploy this on the internal website for a review?

It is already.

File: source/qan-intro.rst (hunk 1 above)
Comment (lunaticusgreen) [reply source/qan-intro.rst:163]: @PaulJacobs-percona https://github.com/percona/grafana-dashboards/blob/e348eefe9cd03cae8a9e5c10e18dc5760cee276a/pmm-app/src/pmm-qan/panel/QueryAnalytics.constants.ts#L41 PTAL here, all descriptions that we have on a frontend

**Comment by rnovikovP** [comment]
> > @PaulJacobs-percona can we deploy this on the internal website for a review?
> 
> It is already.

 I was too fast in the first place (added comment without seeing the link ) and too late now .. the  URL is  overwritten with old content  (

**Pull request merged**